libraries/drivers_framework_rails.rb

```diff
@@ -38,13 +38,14 @@ def database_yml(db_driver)
 
         database = db_driver.out
         deploy_environment = deploy_env
+        replica = node['deploy'][app['shortname']]['sidekiq_on_replica']
```

Comment:
this is not only for sidekiq, let's give it more generic name, i.e `replica_db_settings`

Reply:
this comment is not addressed

Reply:
I've update the variable name and I think we don't need to update the database.yml template.

<img width="1396" alt="Screen Shot 2022-11-21 at 10 59 49" src="https://user-images.githubusercontent.com/301512/202962793-2c66745d-ae35-4292-9289-abd37c1b3bb0.png">

Reply:
I mean variable that we pull from stack settings, is called `sidekiq_on_replica`.

Reply:
![image](https://user-images.githubusercontent.com/10468452/202975319-11f5f6c5-80c3-49b9-833e-6aa0080575d6.png)

Reply:
ah ok got it.. Thank you..